package.json

```diff
@@ -7,6 +7,11 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "cypress": "cypress open",
+    "cypress:run": "cypress run",
+    "e2e": "start-server-and-test dev http://localhost:3000 cypress",
+    "test:e2e": "start-server-and-test dev http://localhost:3000 cypress:run",
```

Comment:
What is the difference between cypress/e2e and cypress:run/test:e2e?

Reply:
it is a little overkill but the idea is to make it easier to run. 
    "cypress": "cypress open",  - this need you to have 2 terminal open, one with the app running and then you open the cypress dev app 
    "cypress:run": "cypress run", this also needs the app to already be running but will run the tests heedlessly 
    "e2e": "start-server-and-test dev http://localhost:3000 cypress", this will auto run the app and then open the test runner
    "test:e2e": "start-server-and-test dev http://localhost:3000 cypress:run", this will auto start the app and run the tests headlessly

Reply:
I find it a bit overkill but if it helps then it is fine. I rather see us use what the github actions uses to keep us consistent